cluster/providers/openstack/utils.sh

```diff
@@ -155,4 +155,40 @@ function packstack_update_endpoints {
     while IFS=" " read -r id url;     do 
         openstack endpoint set --url ${url/127.0.0.1/"$new_host"} $id
     done  < /tmp/output
-}
\ No newline at end of file
+}
+
+function test_snapshot_creation {
+
+    openstack volume snapshot create cirros-volume --force
+    #verify with - openstack volume snapshot create cirros-volume --force
+    /etc/cinder/cinder.conf
```

Comment:
unused?

Reply:
since its not something that trivial , I've added test packstack_test_snapshot_creation , so we  will fail early..